src/plugins/vis_type_timeline/server/lib/config_manager.ts

```diff
@@ -35,11 +35,14 @@ import { configSchema } from '../../config';
 export class ConfigManager {
   private opensearchShardTimeout: number = 0;
   private graphiteAllowedUrls: string[] = [];
+  /** @deprecated use graphiteDeniedIPs */
+  private graphiteBlockedIPs: string[] = [];
```

Comment:
Can we just replace here instead of deprecate as these are private?

Reply:
replaced